src/cljc/rems/text.cljc

```diff
@@ -125,40 +136,36 @@
 
 (defn localize-event [event]
   (let [event-type (:event/type event)]
-    (text-format
-     (get event-types event-type :t.applications.events/unknown)
-     (application-util/get-member-name (:event/actor-attributes event))
+    (str
```

Comment:
I think instead of `str` all the bits together a better design would use clear slots either with `text-format` (because languages could vary) or more hiccup here.

Reply:
Unfortunately localize-event needs to return a string because it's also used in the pdf code. I was thinking of having something like `"%1 approved the application. %2" as the format string for the approved event, but that didn't seem useful.